main/templates/base/fetc_form_base.html

```diff
@@ -31,37 +31,39 @@
                     </div>
                 {% endblock %}
 
-                {{ form }}
-                {% block post-form-pre-buttons %}{% endblock %}
+                {% block auto-form-render %}{{ form }}{% endblock %}
 
-                <div class="uu-form-text-row">
-                    <div class="uu-form-text">
-                        <div class="mt-4 mb-3 w-100 d-flex">
-                            {% block form-buttons %}
-                                {% include "base/form_buttons.html" %}
-                            {% endblock %}
+                {% block form-buttons %}
```

Comment:
I think something went wrong when merging. This template now has two `{% block form-buttons %}` (see below, line 42) which causes an error.